### ISSUE TITLE
fail to train properly

### ISSUE THREAD

**Issue by cppppp** [ltkong218/FastFlowNet#29]
Hi, I have used the code of irr-pwc for training fastflownet, but the loss is very high. I have tried another network that does not use SpatialCorrelationSampler, and it trained properly. I have editted the output of fastflownet:
```
        output_dict = {}
        if self.training:
            output_dict['flow2'] = flow2 * 20.0
            output_dict['flow3'] = flow3 * 20.0
            output_dict['flow4'] = flow4 * 20.0
            output_dict['flow5'] = flow5 * 20.0
            output_dict['flow6'] = flow6 * 20.0
        else:
            output_dict['flow1'] = self.upsample2d_as(flow2, img1) * 20.0
        return output_dict
```
I suspect that the problem is with SpatialCorrelationSampler (version: 0.4.0). Hope for some clues for finding the problem, thanks!
